app/lib/constants/site_config.rb

```diff
@@ -34,7 +34,7 @@ module SiteConfig
         placeholder: "Campaign sidebar enabled or not"
       },
       campaign_sidebar_image: {
-        description: "https://image.url",
+        description: "https://image.url/image.png",
```

Comment:
The way `https://image.url/image.png` reads feels strange to me. Should we make this https://url/image.png or https://image_url/image.png? This is completely personal preference with no rationale so feel free to ignore if you or others don't feel the same way..

Reply:
@Ridhwana I agree with you, tbqh. When I read this, I'm not sure what `image.url/image.png` actually _means_ in terms for formatting 🤷

Reply:
Thanks for the feedback, @Ridhwana! I've made your suggested change in this PR. 😊